### PR TITLE
Add layout_snake

### PR DESCRIPTION
**Pull request by YegorKochetkov**
https://YegorKochetkov.github.io/layout_snake/
https://YegorKochetkov.github.io/layout_snake/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+
+  height: 300px;
+  min-width: 300px;
```
Comment (ivan-kadykalo) [inline src/style.css:17]: Use `flex-basis` instead of `min-width`

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  }
+
+  .block--6 {
+    order: 4;
+  }
+}
+
+@media (min-width: 900px) {
+  .block {
+    width: 33.33%;
```
Comment (ivan-kadykalo) [inline src/style.css:72]: will be better to use `calc(100% / 3)` and `flex-basis`

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  background-color: rgb(51, 0, 0);
+}
+
+.block--6 {
+  background-color: #000;
+}
+
+@media (min-width: 600px) {
+  .block {
+    width: 50%;
```
Comment (ivan-kadykalo) [inline src/style.css:50]: here also

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Nice code))
Only couple things left

**Comment by ivan-kadykalo** [comment]
For some reasons your links doesn't work. 
image

**Review by MaksimusCRS — APPROVED** [review]
GJ!

**Pull request closed without merging**